go.mod

```diff
@@ -47,6 +47,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
```

Comment:
Please rename the PR according to git commit [style guide](https://github.com/moira-alert/moira/blob/master/.github/CONTRIBUTING.md#git-style-guide)

Reply:
fixed